lib/configuration/variables/sources/instance-dependent/get-ssm.js

```diff
@@ -20,6 +20,8 @@ module.exports = (serverlessInstance) => {
         Error: ServerlessError,
         errorMessage: 'Non-string address argument in variable "ssm" source: %v',
       });
+      const region = !params || !params[0] || params[0] === 'raw' ? null : params[0];
```

Comment:
Looks like this change is causing an error for me:

```
Cannot resolve serverless.yml: Variables resolution errored with:
    - Cannot resolve variable at "<var name>": Missing region in config
```

I think this logic is resulting in `region = null`, but it used to resolve to `false`?

I haven't dug much deeper, but I was able to work around the error by adding the region param, ie. `ssm(us-east-1):...`

Reply:
@rschick great thanks for pointing. Will be fixed with https://github.com/serverless/serverless/pull/9236